org.openhab.binding.zigbee/ESH-INF/thing/channels.xml

```diff
@@ -146,6 +146,15 @@
         <state readOnly="true" />
     </channel-type>
 
+    <!-- IAS Tamper Channel -->
+    <channel-type id="ias_tamper">
+        <item-type>Switch</item-type>
+        <label>Tamper</label>
+        <description></description>
+        <category></category>
```

Comment:
Would `Alarm` be a suitable category here? (From the bottom of https://www.eclipse.org/smarthome/documentation/concepts/categories.html).

Do you have any opinion here, @cdjackson ?